controllers/utils/env.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func GetEnv(key string, filenames ...string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	envMap, err := godotenv.Read(filenames...)
```

Comment:
What happens in case of multiple files having same env variable but different values ? Would it read sequentially and map the last known value ?

Reply:
I believe so. Values get merged into a map.